controllers/articles.js

```diff
@@ -143,6 +151,47 @@ const articles = {
         });
       }
 
+      // tag registration
+      const oldTags = findArticle.tagList;
+      let { newTags } = req.body;
```

Comment:
@fridolinho  **newTags** in `req.body` doesn't exist! When we update we pass **tagList** not _**newTags**_ So it is still failling with content-type _application/x-www-from-urlencoded_